.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "git.ignoreLimitWarning": true
+}
```

Comment:
VS Code rules should not be committed to the repo, try adding a rule to the .gitignore file so that the .vscode folder is ignored.